composer.json

```
@@ -15,10 +15,18 @@
   },
   "require": {
     "php": "^7.0",
-    "yiisoft/yii2": ">=2.0.0 <2.1.0"
+    "yiisoft/yii2": "^2.0.0",
+    "npm-asset/quill": "^1.3.7"
   },
   "require-dev": {
-    "roave/security-advisories": "dev-master"
+    "roave/security-advisories": "dev-master",
+    "yiisoft/yii2-coding-standards": "^2.0.0",
+    "npm-asset/highlightjs": "^9.16.2",
```

`highlightjs` is deprecated apparently. Let's use:

```suggestion
    "npm-asset/highlight.js": "^9.18.1",
```